### PR TITLE
Added meditation feature

### PR DESCRIPTION
**Pull request by shrey2003**
### 👨‍💻 Changes proposed
Added meditation feature files.
This feature allows user to meditate by selecting the required duration and required environment(forest, rain, summer night, beach) and sound effect. This project is created using HTML, CSS, JavaScript .

### ✅ Check List (Check all the applicable boxes)

- [✅] My code doesn't break any part of the project (Zero Octave-JavaScript- Projects).
- [✅] This PR does not contain plagiarized content.
- [✅] My Addition/Changes works properly and matches the overall repo pattern.
- [✅] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

image

### CODE REVIEW

File: Public/meditation.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+             </button>
+             <button data-sound="../assets/Audio/meditation/forest.mp3" data-video="../assets/Video/meditation/forest.mp4">
+                <img src="../assets/Images/meditation/forest.jpg" >
+            </button>
+            <button data-sound="../assets/Audio/meditation/summer night.mp3" data-video="../assets/Video/meditation/summer night.mp4">
+              <img src="../assets/Images/meditation/night.jpg" >
+            </button>
+         </div>
+     </div>
+    <script src="app.js"></script>
```
Comment (NOiR-07) [inline Public/meditation.html:52]: give the correct directory of the js folder

**Comment by shrey2003** [comment]
@NOiR-07 my issue number is #264

**Comment by NOiR-07** [comment]
fix the indentation of your card on website

**Comment by shrey2003** [comment]
@NOiR-07 indentation of card corrected

**Comment by NOiR-07** [comment]
there is some conflict please checkout

**Pull request merged**